hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockStreamConfig.java

```diff
@@ -44,4 +51,8 @@ public record BlockStreamConfig(
         @ConfigProperty(defaultValue = "32") @NetworkProperty int hashCombineBatchSize,
         @ConfigProperty(defaultValue = "1") @NetworkProperty int roundsPerBlock,
         @ConfigProperty(defaultValue = "localhost") String grpcAddress,
-        @ConfigProperty(defaultValue = "8080") @Min(0) @Max(65535) int grpcPort) {}
+        @ConfigProperty(defaultValue = "8080") @Min(0) @Max(65535) int grpcPort,
+        @ConfigProperty(defaultValue = "3") @NetworkProperty int uploadRetryAttempts,
+        @ConfigProperty(defaultValue = "168") @NetworkProperty int localRetentionHours,
```

Comment:
Just a thought, `@NodeProperty` may be more appropriate for these. Not sure and not critical